source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md

```diff
@@ -0,0 +1,65 @@
+---
+title: vagrant-service-manager 1.3.0 released focusing on Kubernetes support
+author: budhram
+date: 2016-08-16 12:54:00 UTC
+tags: vagrant, devtools, releases, kubernetes
+published: true
+comments: true
+---
+
+This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes setup information which enable users to access Kubernetes server running inside ADB VM from host.
```

Comment:
This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes configuration information.  This enable users to access the Kubernetes server running inside ADB virtual machine from their host.